bom/application/pom.xml

```diff
@@ -4186,6 +4186,14 @@
                         <groupId>commons-logging</groupId>
                         <artifactId>commons-logging</artifactId>
                     </exclusion>
+                    <exclusion>
+                        <groupId>org.jboss.resteasy</groupId>
+                        <artifactId>resteasy-client</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.jboss.resteasy</groupId>
+                        <artifactId>resteasy-jackson2-provider</artifactId>
+                    </exclusion>
```

Comment:
These two resteasy dependencies were added in 5.12.1:
https://mvnrepository.com/artifact/org.jboss.narayana.rts/lra-client/5.12.0.Final
https://mvnrepository.com/artifact/org.jboss.narayana.rts/lra-client/5.12.1.Final
(and were updated in 5.12.2)

As far as I understood #21172, users shall have the choice which resteasy flavor they prefer, so excluding classic resteasy made sense to me. Also, without these exclusions there will be a banned dependency to jakarta-activation.
/cc @geoand

Reply:
Right, that is what https://github.com/quarkusio/quarkus/pull/21172 does